CONTRIBUTING.md

```diff
@@ -19,6 +19,11 @@ You can contribute by:
 All bugs, tasks or enhancements are tracked as GitHub issues.
 Issues which might be a good start for new contributors are marked with the “good-start” label.
 
+## Code Review
+
+All contributions must be reviewed by at least one Code Owner before being merged.  If a contribution
+is made by someone who is already a Code Owner, that contribution must be reviewed by at least one other Code Owner.
```

Comment:
Given the definition of contribution in the section above one can read this as saying we require a code owner to review all issues before accepting them, I don't think thats your intention. 

How about something more like:
```suggestion
All changes to which are to be included in project source control are required to be reviewed by at least one Code Owner in addition to the changes author prior to being merged.  
```

Reply:
We are in a block called `code review` is it safe enough to call it `code changes`? Also the revised sentence has quite a lot of logic in it. I think it needs two sentences or parentheses to help make it readable.

```
All code changes must be reviewed by at least one Code Owner before being merged.  If the author of a change is a Code Owner, then at least one other Code Owner must review that change.
```

Reply:
I wondered about code changes but , it applies equally to docs or blog posts in the website repo which is why I went for the more general language.

I have no objections to two using two sentences however.

Reply:
I've reworded.  I agreed with @SamBarker that the word contributions was ambiguous, and like @robobario I prefered the two sentences.